docs/pbc/all/tax-management/202410.0/base-shop/third-party-integrations/vertex/install-vertex/install-vertex.md

```diff
@@ -37,3 +37,4 @@ To install Vertex, integrate the ACP connector module and the Vertex app:
 
 1. [Integrate ACP connector module for tax calculation](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-acp-connector-module-for-tax-calculation.html)
 2. [Integrate the Vertex app](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.html)
+3. [Integrate the Taxamo app](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-taxamo-app.html)
```

Comment:
Is this not optional?

Reply:
it's not. we have one configuration for the Vertex. but if we don't create an integration for Taxamo we'll have a configuration for it (it's optional) but it won't be possible to use from the SCOSS side. in this case it's require for vertax integration